packages/mesos/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "d5746045ac740d5f28f238dc55ec95c89d2b7cd9",
-    "ref_origin" : "dcos-mesos-1.0.x-837bb4e"
+    "ref": "fd011fcce61ecf4c4d4cc3cde09a11c1470ff1b6",
```

Comment:
I assume we will eventually want to bump Mesos to 1.1.0 or beyond, regardless of when the pod feature is shipped.

I can take this commit out if I tweak some of the module code (i.e. use an older version of the interfaces).